Plots/Masking/NCL_mask_5.py

```diff
@@ -0,0 +1,120 @@
+"""
+NCL_mask_5.py
+=============
+This script illustrates the following concepts:
+    - Using cartopy.feature options to display map information
+    - Paneling two plots on a page with 'subplots' command
+    - Drawing contours
+    - Explicitly setting the fill colors for contours
```

Comment:
I don't think this concept is illustrated in this example anymore. You could reword this to refer to setting the fill colors for the map features on the first plot, or remove this bullet all together.